physicalplan/scan/function.go

```diff
@@ -70,6 +70,16 @@ func NewFunctionCall(f *parser.Function, selectRange time.Duration) (FunctionCal
 				Metric: labels,
 			}
 		}, nil
+	case "last_over_time":
+		return func(labels labels.Labels, points []promql.Point, stepTime time.Time) promql.Sample {
+			return promql.Sample{
+				Point: promql.Point{
+					T: stepTime.UnixMilli(),
+					V: points[len(points)-1].V,
```

Comment:
Do we need to check for out of bounds here? Will `points` always be a slice with at least one element?

Reply:
Good question, I think that's something we need to handle.

Reply:
Umm, is it possible to have the series with labels only but no sample? I guess no so this should be safe. If no series then the func won't be called

Reply:
Maybe it can happen if there are gaps in the values, but I also need to double check if this is a real problem.